figures/scripts/fig4-tp53-telomerase-panels.R

```diff
@@ -187,6 +198,13 @@ plot_tp53 <- function(df, pvalue_y) {
       axis.ticks = element_line(size = rel(0.5))
     )
   
+  return(
+    list(
+      "plot" = df_counts_plot,
+      "df" = df_counts
+    )
+  )
+  
```

Comment:
I'm looking at this early, but I just had the thought that I probably wouldn't have done this... instead you could just use `tp53_scores_plot$data`, etc. to get the data back out of the `ggplot2` object. (This may save you time in future edits)

Reply:
Hindsight is a beautiful thing... Next time :)